languages/swift/exercises/concept/tuples/.meta/Example.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+func cartesianToPolar(_ cart: (x: Double, y: Double)) -> (r: Double, theta: Double) {
+  (r: (cart.x * cart.x + cart.y * cart.y).squareRoot(), theta: atan2(cart.y, cart.x))
+}
+
+func combineRecords(
+  production: (toy: String, id: Int, productLead: String),
+  gifts: (Int, [String])
+) -> (id: Int, toy: String, productLead: String, recipients: [String]) {
+  return (id: production.id, toy: production.toy, productLead: production.productLead, gifts.1)
```

Comment:
see my comment in instruction.md: Maybe add some handling of different ids here.